src/bindings/c/tests/CMakeLists.txt

```diff
@@ -44,6 +44,46 @@ set(TARGET_NAME "ov_capi_test")
 file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/ov_*.cpp)
 file(GLOB HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 
+find_package(OpenCL QUIET)
+
+if(NOT OpenCL_FOUND)
+    MESSAGE(WARNING "OpenCL is disabled or not found, ${TARGET_NAME} will be built without OpenCL support. Install OpenCL.")
+endif()
+
+find_path(OpenCL_HPP_INCLUDE_DIR
+    NAMES
+    CL/cl2.hpp OpenCL/cl2.hpp
+    HINTS
+    ${opencl_root_hints}
+    ENV "PROGRAMFILES(X86)"
+    ENV AMDAPPSDKROOT
+    ENV INTELOCLSDKROOT
+    ENV NVSDKCOMPUTE_ROOT
+    ENV CUDA_PATH
+    ENV ATISTREAMSDKROOT
+    ENV OCL_ROOT
```

Comment:
Do we really need to take a look across so big set of different places?

Reply:
It follows the same method with benchmark_app to find opencl dependencies, and we can remove some of them that are not for intel opencl.

Reply:
done